build.gradle

```diff
@@ -86,11 +90,14 @@ jacocoTestReport {
 }
 
 //NOTE: we ignore contracts for now
+// TEMPORARY HACK: JP removed -Werror because
+// I'm using the latest version of htsjdk and get a depreciation warning
+// (presumably the latest version of gatk fixes this)
```

Comment:
Let's not do this -- I'll open a separate PR against master to fix any deprecation issues now and move master to the same snapshot you're using, and when you rebase be sure to restore `-Werror`